crates/iota-data-ingestion-core/src/executor/mod.rs

```diff
@@ -0,0 +1,6 @@
+// Copyright (c) 2025 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+pub(crate) mod v1;
```

Comment:
All `pub` components of the curernt `v1` that was also `pub` before, should be exposed to maintain backward compatibility.